README.md

```diff
@@ -299,7 +299,76 @@ The default XML parser contained a security risk which made the driver prone to
 
 | Parameter       | Value           | Required      | Description  |
 | ------------- |:-------------:|:-------------:| ----- |
-|`allowXmlUnsafeExternalEntity` | Boolean | No | Set to true if you would like to use XML inputs that refer to external entities. WARNING: Setting this to true is unsafe since your system to be prone to XXE attacks.<br/><br/>**Default value:** `false` | 
+|`allowXmlUnsafeExternalEntity` | Boolean | No | Set to true if you would like to use XML inputs that refer to external entities. WARNING: Setting this to true is unsafe since your system to be prone to XXE attacks.<br/><br/>**Default value:** `false` |
+
+### AWS IAM Authentication
+
+An optional authentication method is to use Amazon AWS Identity and Access Management (IAM). 
+When using AWS IAM Authentication, Connection URL must be an Amazon endpoints, and not a custom domain or IP Address. 
```

Comment:
```suggestion
When using AWS IAM authentication, Connection URL must be a valid Amazon endpoint, and not a custom domain or an IP address. 
```
Can you please provide some example URLs?

Reply:
db-identifier.cluster-XYZ.us-east-2.rds.amazonaws.com